packages/kit/src/vite/dev/index.js

```diff
@@ -212,6 +213,23 @@ export async function dev(vite, svelte_config) {
 					}
 				}
 
+				const file = posixify(path.resolve(decoded.slice(1)));
+
+				// temporary — debugging windows via CI
+				if (file.includes('logo.svg')) {
+					console.error({ file, allowed });
+				}
+
+				if (
+					fs.existsSync(file) &&
+					!fs.statSync(file).isDirectory() &&
+					allowed.some((dir) => file.startsWith(dir))
```

Comment:
if we're going to do this we also need to check if `server.fs.strict` is enabled

Reply:
done